test/unit_tests/braket/circuits/test_gates.py

```diff
@@ -345,6 +345,18 @@ def test_ir_gate_level(testclass, subroutine_name, irclass, irsubclasses, kwargs
             OpenQASMSerializationProperties(qubit_reference_type=QubitReferenceType.PHYSICAL),
             "i $4;",
         ),
+        (
+            Gate.CY(),
+            [4, 4],
```

Comment:
nit: a gate can't be controlled on the same qubit it acts on. While testing coverage is the same, it would be slightly more realistic if this list was two different qubits

Reply:
That's fair! I'll change it up then.